Assets/Scripts/Controllers/MainMenuController.cs

```diff
@@ -0,0 +1,63 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+
+using UnityEngine;
+using UnityStandardAssets.Utility;
+
+public class MainMenuController : MonoBehaviour
+{
+    public ModsManager modsManager;
+
+    public static MainMenuController Instance { get; protected set; }
+
+    // Use this for initialization.
+    public void OnEnable()
+    {
+        new PrototypeManager();
+    }
+
+    public void Start()
+    {
+        Instance = this;
+
+        new SpriteManager();
+        new AudioManager();
+        modsManager = new ModsManager();
```

Comment:
Should we load these here or in the GameController? I think that it might be clear to just leave this to deal with the MainMenu and not with loading stuff. By "these" I mean PrototypeManager, SpriteManager, AudioManager and ModsManager.